src/pages/LeaderboardsPage.tsx

```diff
@@ -53,129 +53,120 @@ export const LeaderboardsPage = () => {
   return (
     <>
       <Box
-        width="100%"
-        position={{ sm: "absolute" }}
-        px={0}
-        top={0}
-        left={0}
-        zIndex={-1}
+        height={211}
```

Comment:
for later but this will prob need to get changed to a responsive value (similar to what james did for the footer)